src/components/plot.jsx

```diff
@@ -0,0 +1,32 @@
+import {CartesianGrid, Line, LineChart, ResponsiveContainer, Tooltip, XAxis, YAxis} from "recharts"
+import React from "react";
+import PropTypes from 'prop-types';
+
+function Plot({key, plotData}) {
+    return (
+        <div style={{display: 'flex', justifyContent: 'center', alignItems: 'center', height: '100vh'}}>
+            <ResponsiveContainer width={1000} aspect={2.5}>
+                <LineChart data={plotData.points}>
+                    <YAxis/>
+                    <CartesianGrid stroke="#eee" strokeDasharray="5 5"/>
+                    <XAxis dataKey={"timestamp"} hide={false}/>
```

Comment:
Many hardcoded "magic" values (like height and width) - I think we should extract them somewhere, unless there is a solid reason not to (in that case pls explain).